code/modules/jobs/job_types/security_cadet.dm

```diff
@@ -0,0 +1,67 @@
+/datum/job/security_cadet
+	title = JOB_SECURITY_CADET
+	description = "Maintain order, detain space criminals with the Security Officers, \
+		help Warden with securing and proccesing prisoners, \
+		accidentaly pepperspray yourself."
+	auto_deadmin_role_flags = DEADMIN_POSITION_SECURITY
+	department_head = list(JOB_HEAD_OF_SECURITY)
+	faction = FACTION_STATION
+	total_positions = 3
+	spawn_positions = 3
+	supervisors = "the security department and Head of Security"
+	minimal_player_age = 7
```

Comment:
this means you need to have played 7 days, you sure you want that for this?

Reply:
No, I forgot to delete this